packages/browser/src/BacktraceBrowserRequestHandler.ts

```diff
@@ -18,14 +22,8 @@ export class BacktraceBrowserRequestHandler implements BacktraceRequestHandler {
     private readonly MULTIPART_HEADERS = {
         'Transfer-Encoding': 'chunked',
     };
-    constructor(
-        private readonly _options: {
-            url: string;
-            token?: string;
-            timeout?: number;
-        },
-    ) {
-        this._timeout = this._options.timeout ?? DEFAULT_TIMEOUT;
+    constructor(private readonly _options?: BacktraceBrowserRequestHandlerOptions) {
```

Comment:
small nit: there is no need to keep _options in the class instance. we can remove `private readonly _` since we don't use it anywhere